### DISCUSSION TITLE
Can I create Interactions in MDX or should I convert stories to Component Story Format (CSF)?

### DISCUSSION THREAD

**Discussion by davebrewer** [storybookjs/storybook#24806] [Help]
Currently have stories in MDX and wondering if Interaction is supported or need to convert over to CSF?
Guessing long term CSF is probably the wise move, is that the case?

**Comment by jonniebigodes** [comment]
Thanks for taking the time to reach out to us with your question. We appreciate it. Writing stories and interactions are deprecated as of Storybook 7.0, and we're encouraging users to write their stories and interaction tests in story files and leaving the MDX format for custom documentation for components.

Looking forward to hearing how the transition from MDX-based stories to CSF stories goes, and feel free to reach out if you run into any complications.

Have a fantastic day.

Stay safe
